generator/sig_readme.tmpl

```diff
@@ -2,18 +2,18 @@
 # {{.Name}} SIG
 
 {{ .MissionStatement }}
-## Meetings
+{{ if .Meetings }}## Meetings
 {{- range .Meetings }}
-* [{{.Day}}s at {{.Time}} {{.TZ}}]({{$.MeetingURL}}) ({{.Frequency}}). [Convert to your timezone](http://www.thetimezoneconverter.com/?t={{.Time}}&tz={{.TZ | tzUrlEncode}}).
+* {{.Description}}: [{{.Day}}s at {{.Time}} {{.TZ}}]({{.URL}}) ({{.Frequency}}). [Convert to your timezone](http://www.thetimezoneconverter.com/?t={{.Time}}&tz={{.TZ | tzUrlEncode}}).
+{{- if .ArchiveURL }}
+  * Meeting notes and Agenda can be found [here]({{.ArchiveURL}}).
```

Comment:
how do you feel about just changing this to `[Meeting notes and Agenda]({{.ArchiveURL}})`?

Reply:
I like it! *changes*